src/content/en/fundamentals/performance/optimizing-content-efficiency/image-optimization.md

```diff
@@ -2,7 +2,7 @@ project_path: /web/fundamentals/_project.yaml
 book_path: /web/fundamentals/_book.yaml
 
 
-{# wf_updated_on: 2017-07-24 #}
+{# wf_updated_on: 2018-02-16 #}
 {# wf_published_on: 2014-05-06 #}
```

Comment:
Please add `{# wf_blink_components: Blink>Image #}` below this line

Reply:
Added.